src/shared/services/utils.ts

```diff
@@ -312,10 +312,10 @@ export const arrayToObject = (array: any) =>
 export const stringifyMod = (
   value: any,
   modFn: any = null,
-  pretty: any = false,
+  pretty: boolean | number = false,
   skipOpeningIndentation = false
-): any => {
-  const prettyLevel = !pretty ? false : pretty === true ? 1 : parseInt(pretty)
+): string => {
+  const prettyLevel = !pretty ? false : pretty === true ? 1 : +pretty
```

Comment:
Thinking this'd be easier to follow as `  const prettyLevel = isNumber(pretty) ? pretty : pretty ? 1 : 0`

but the whole method is pretty messy anyhow